CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.2.1] - 2017-02-08
```

Comment:
sure, thats kind of what I was expecting, but the developer guidelines asked for a Changelog entry with a PR.